examples/example-server-lib/floating_window_manager.cpp

```diff
@@ -16,16 +16,22 @@
 
 #include "floating_window_manager.h"
 #include "decoration_provider.h"
+#include "mir_toolkit/common.h"
+#include "miral/minimal_window_manager.h"
+#include "miral/window_specification.h"
```

Comment:
These are not local headers, so use `<...>` includes (and they belong in the block of Mir headers)